doc/source/whatsnew/v1.5.0.rst

```diff
@@ -989,6 +989,7 @@ Indexing
 - Bug in :meth:`DataFrame.sum` min_count changes dtype if input contains NaNs (:issue:`46947`)
 - Bug in :class:`IntervalTree` that lead to an infinite recursion. (:issue:`46658`)
 - Bug in :class:`PeriodIndex` raising ``AttributeError`` when indexing on ``NA``, rather than putting ``NaT`` in its place. (:issue:`46673`)
+- Bug in :meth:`DataFrame.loc` when enlarging a :class:`Series` with dtype :class:`CategoricalDtype` with a scalar (:issue:`47677`)
```

Comment:
From the example in https://github.com/pandas-dev/pandas/issues/47677#issue-1301787672, wasn't this just a regression on main (i.e. no whatsnew needed)?

Reply:
Good point, missed this when checking out the branch, removed